lib/pages/dashboard/blocs/agenda_bloc.dart

```diff
@@ -44,6 +44,15 @@ class AgendaLoadWeek extends AgendaEvent {
   List<Object> get props => [week];
 }
 
+class AgendaLoadTeamMonth extends AgendaEvent {
```

Comment:
```suggestion
class LoadTeamMonthAgenda extends AgendaEvent {
```
When naming an event start with a verb as it's an action